docker/push

```diff
@@ -91,4 +91,11 @@ then
 else
   echo "No docker credentials provided. Skip uploading ${IMAGE_REPO} to quay";
 fi;
+#Push image to run openebs-e2e based on git commit
+if [ ! -z "${COMMIT}" ];
```

Comment:
check like this would be good for GITLAB_DNAME and GITLAB_DPASS:
`if [ ! -z "${QNAME}" ] && [ ! -z "${QPASS}" ];` and an error message in 'else' case